src/middlewared/middlewared/utils/acltools.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding=utf-8 -*-
+import enum
+from bsd import acl
+
+"""
+These are simplified forms of permissions sets based
+on NTFS basic permissions.
+"""
```

Comment:
Do you imagine us ever using this outside filesystem plugin? If not embedding this in the plugin makes sense to me.

Reply:
That's a good point. Cleaning this up.